mne/io/fiff/tests/test_raw.py

```diff
@@ -115,6 +115,33 @@ def test_copy_append():
     assert_equal(data.shape[1], 2 * raw._data.shape[1])
 
 
+@testing.requires_testing_data
```

Comment:
you also don't need the testing data for this, use `test_fif_fname` instead, that's included with the repo